account_invoice_pricelist_sale/models/sale_order.py

```diff
@@ -0,0 +1,15 @@
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _prepare_invoice(self):
+        """Pricelist_id is set on invoice."""
+        self.ensure_one()
+        val = super(SaleOrder, self)._prepare_invoice()
```

Comment:
Apply PEP 3135 (28-Apr-2007) to "super()."
```suggestion
        val = super()._prepare_invoice()
```

Reply:
updated